Tests/Client/Unit Tests/AtomicTests.swift

```diff
@@ -164,13 +195,15 @@ extension AtomicTests {
             }
 
             for _ in 0...5 {
-                DispatchQueue.random.async {
+                // We need to capture the value explicitly. Without this the tests randomly crashes after
+                // `atomicValue` is assigned to `nil`.
+                DispatchQueue.random.async { [atomicValue] in
```

Comment:
10 minutes to identify the problem + 50 minutes figuring out how to fix it = `[atomicValue]`